cpp/CPP14Parser.g4

```diff
@@ -329,8 +329,8 @@ declSpecifier:
 	| Friend
 	| Typedef
 	| Constexpr;
-
-declSpecifierSeq: declSpecifier+ attributeSpecifierSeq?;
+	
+declSpecifierSeq: declSpecifier+? attributeSpecifierSeq?;
```

Comment:
Why just not `declSpecifier*` instead of `declSpecifier+?`

Reply:
Because we need at least one.